explorer/trace_testdata/full_trace.carbon

```diff
@@ -93,18 +93,18 @@ fn Main() -> i32 {
 // CHECK:STDOUT: checking BindingPattern n: i32
 // CHECK:STDOUT: checking ExpressionPattern i32
 // CHECK:STDOUT: checking IntTypeLiteral i32
-// CHECK:STDOUT: (+) stack-push: i32 .0.
-// CHECK:STDOUT: (+) stack-push: i32 .0.
+// CHECK:STDOUT: (+) stack-push: pos-0 ValueExpressionAction `i32` (full_trace.carbon:11)
```

Comment:
It seems really surprising to me to print `pos` before the action kind. `pos` (and the rest of the information) isn't meaningful until you know the kind, so it really seems like the kind should go first.

Reply:
Oh okay, I'll move it after `ActionKind` in the next commit.